tests/unit_tests/forward_model_runner/test_job.py

```diff
@@ -40,6 +41,48 @@ def test_run_with_process_failing(
         next(run)
 
 
+@pytest.mark.integration_test
+def test_cpu_seconds_can_detect_multiprocess():
+    """Run a job that sets of two simultaneous processes that
+    each run for 1 second. We should be able to detect the total
+    cpu seconds consumed to be roughly 2 seconds."""
+    pythonscript = "busy.py"
+    with open(pythonscript, "w", encoding="utf-8") as pyscript:
+        pyscript.write(
+            textwrap.dedent(
+                """\
+            import time
+            now = time.time()
+            while time.time() < now + 1:
+                pass"""
+            )
+        )
+    scriptname = "saturate_cpus.sh"
+    with open(scriptname, "w", encoding="utf-8") as script:
+        script.write(
+            textwrap.dedent(
+                """\
+            #!/bin/sh
+            python busy.py &
+            python busy.py"""
+            )
+        )
+    executable = os.path.realpath(scriptname)
+    os.chmod(scriptname, stat.S_IRWXU | stat.S_IRWXO | stat.S_IRWXG)
+    job = Job(
+        {
+            "executable": executable,
+        },
+        0,
+    )
+    job.MEMORY_POLL_PERIOD = 0.1
+    cpu_seconds = 0.0
+    for status in job.run():
+        if isinstance(status, Running):
+            cpu_seconds = max(cpu_seconds, status.memory_status.cpu_seconds)
+    assert 1.6 < cpu_seconds < 2.4
```

Comment:
There might be good reasons for checking against such a large range. Is it really that inaccurate?
We can consider increasing the test length slightly if that helps?

Reply:
It is merely a first guesstimate at what could mitigate flakyness..

Reply:
On MacOS:

`FAILED tests/unit_tests/forward_model_runner/test_job.py::test_cpu_seconds_can_detect_multiprocess - assert 1.6 < 1.23128864
`
🤔

Reply:
Passed on a rerun, so yep, there is already flakyness here  😢

Reply:
Maybe add a flaky marker to rerun on failures? Seems reasonable as we are dealing with timing issues here.